qiskit/providers/aer/openpulse/readme.md

```diff
@@ -56,4 +80,3 @@ frame that a resonant pulse is close to DC.
 
 The measurement operators are the projections onto the 1 excitation subsystem for qubit `l`
```

Comment:
Can you add that "for measurement level 2 the simulator outputs counts, for level 1 it outputs 0 and 1 as floats" (or something like this)